src/ayb_db/models.rs

```diff
@@ -173,6 +173,37 @@ pub struct Entity {
     pub links: Option<Vec<Link>>,
 }
 
+#[derive(Debug, Serialize, Deserialize)]
+pub struct PartialEntity {
+    pub slug: Option<String>,
+    pub entity_type: Option<i16>,
+    pub display_name: Option<Option<String>>,
+    pub description: Option<Option<String>>,
+    pub organization: Option<Option<String>>,
+    pub location: Option<Option<String>>,
+    pub links: Option<Option<Vec<Link>>>,
+}
+
+impl Default for PartialEntity {
```

Comment:
I didn't know about this --- I'm going to try to use the same approach on my WIP for #251! :)